Modules/_datetimemodule.c

```diff
@@ -1047,6 +1047,24 @@ new_datetime_ex(int year, int month, int day, int hour, int minute,
                             tzinfo, 0, type);
 }
 
+static PyObject *
+new_datetime_ex_capi(int year, int month, int day, int hour, int minute,
```

Comment:
I don't think that "_ex" is useful. Just rename it to "new_datetime_capi", no? Same remark for other functions.

Reply:
Your suggestion includes non-capi functions?